instrumentation/active_record/Gemfile

```
@@ -12,5 +12,5 @@ group :test do
   gem 'byebug'
   gem 'opentelemetry-instrumentation-base', path: '../base'
   gem 'pry-byebug'
-  gem 'sqlite3-ruby'
+  gem 'sqlite3', '~> 1.4.2'
```

Mind if we allow minor version bumps?

```suggestion
  gem 'sqlite3', '~> 1.4'
```